engine/src/main/java/io/zeebe/engine/processor/StreamProcessor.java

```diff
@@ -186,7 +186,7 @@ private long recoverFromSnapshot() {
           String.format(ERROR_MESSAGE_RECOVER_FROM_SNAPSHOT_FAILED, snapshotPosition, getName()));
     }
 
-    LOG.info(
+    LOG.debug(
```

Comment:
I personally would keep the level at info for this one, as it indicates a state transition of the broker processing.